index.html

```diff
@@ -78,74 +79,103 @@ <h5>Skills</h5>
-                <h2 class="second-header">Multi-Post Stories</h2>
-                <p class="second-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
-                <div class="workwith">
-                    <ul class="base">
-                        <li>css</li>
-                        <li>html</li>
-                        <li>bootstrap</li>
-                        <li>Ruby</li>
-                    </ul>
+                <div class="post-3">
+                    <img class="placeholder" src="./images/Img-Placeholder.svg" alt="placeholder">
+                    <div class="post-3-d">
+                        <h2 class="second-header">Multi-Post Stories</h2>
+                        <p class="second-para">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                        <div class="workwith">
+                            <ul class="base">
+                                <li>css</li>
+                                <li>html</li>
+                                <li>bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <button class="butt" type="button">See Project</button>
+                    </div>
                 </div>
-                <button class="butt" type="button">See Project</button>
             </section>
             <footer>
                 <h2 class="contact">CONTACT US</h2>
+                <h2 class="chat">Let's chat!</h2>
                 <p class="third-para">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
                 <form class="form" action="https://formspree.io/f/mrgjldpr" method="post">
-                    <input type="text" class="input" name="name" placeholder="Name (Surname first)" maxlength="30" required>
+                    <input type="text" class="input" name="name" placeholder="Name" maxlength="30" required>
                     <input type="email" class="input" name="email" placeholder="Email" required>
                     <textarea name="textarea" id="textarea" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
                     <button class="butt" type="submit">See Project</button>
                 </form>
+                <form class="form-2" action="https://formspree.io/f/mrgjldpr" method="post">
+                    <section class="last">
+                        <input type="text" class="input" name="first-name" placeholder="First Name" maxlength="30" required>
```

Comment:
- [ ] Please increase the input height for the desktop version. It looks a bit smaller.